package.json

```diff
@@ -11,7 +11,9 @@
     "lint:fix": "npm run lint -- --fix",
     "prettier": "npx prettier src --check",
     "prettier:fix": "npm run prettier -- --write",
-    "format": "npm run prettier:fix && npm run lint:fix"
+    "format": "npm run prettier:fix && npm run lint:fix",
+    "test": "vitest",
```

Comment:
Why not Ava?

Reply:
This just seemed like the path of least resistance with vite/typescript/react, no good reason otherwise. The only thing is I could not use jest because it doesn't work with ses, and we want to directly test AmountMath stuff

Reply:
Reading up on it, it looks pretty great! I missed Jest ergonomics when I started with Ava but I accepted it for the SES requirement. Now that we have another option, maybe Vitest is what we should use in UI packages.